statistics.md

```diff
@@ -338,7 +338,40 @@ To avoid the situation that modifying a small amount of data on a small table fr
 >
 > Currently, the automatic update does not record the configuration items input at manual `ANALYZE`. Therefore, when you use the `WITH` syntax to control the collecting behavior of `ANALYZE`, you need to manually set scheduled tasks to collect statistics.
 
+<<<<<<< HEAD
 Before v5.0, when you execute a query, TiDB collects feedback with `feedback-probability` and updates the histogram and Count-Min Sketch based on the feedback. **Since v5.0, this feature is disabled by default, and it is not recommended to enable this feature.**
+=======
+Before TiDB v5.0, when you execute a query, TiDB collects feedback with `feedback-probability` and updates the histogram and Count-Min Sketch based on the feedback. **Since v5.0, this feature is disabled by default. Since v5.4, this feature has been deprecated, and it is not recommended to enable this feature.**
```

Comment:
```suggestion
Before TiDB v5.0, when you execute a query, TiDB collects feedback with `feedback-probability` and updates the histogram and Count-Min Sketch based on the feedback. **Since v5.0, this feature is disabled by default. Since v5.4, this feature has been deprecated, and it is not recommended to enable this feature.**
```